src/main/controllers/still_working/content.ts

```diff
@@ -0,0 +1,39 @@
+import { isFieldFilledIn } from '../../components/form/validator';
+import { FormContent } from '../../definitions/form';
+import { AnyRecord } from '../../definitions/util-types';
+
+export const stillWorkingContent: FormContent = {
+  fields: {
+    isStillWorking: {
+      classes: 'govuk-radios--inline',
+      id: 'still-working',
+      type: 'radios',
+      label: (l: AnyRecord): string => l.label,
+      values: [
+        {
+          name: 'working',
+          label: (l: AnyRecord): string => l.optionText1,
+          value: 1,
```

Comment:
These values are super misleading. We can't be saving this kind of information to CCD.

Reply:
updated them now to be less misleading